src/helix-ui/elements/HXErrorElement.js

```diff
@@ -1,21 +1,13 @@
 import { HXElement } from './HXElement';
 import shadowStyles from './HXErrorElement.less';
-
-const tagName = 'hx-error';
-const template = document.createElement('template');
-
-template.innerHTML = `
-  <style>${shadowStyles}</style>
-  <hx-icon type="exclamation-circle"></hx-icon>
-  <slot></slot>
-`;
+import shadowMarkup from './HXErrorElement.html';
```

Comment:
The [rollup configuration](https://github.com/rackerlabs/helix-ui/blob/master/rollup.config.js#L17) collapses whitespace when it gets `import`ed.